paasta_tools/check_marathon_services_replication.py

```diff
@@ -26,7 +26,7 @@
 
 After retrieving that information, a fraction of available instances is calculated
 (available/expected), and then compared against a threshold. The default threshold
-is .50, meaning if fewer than 50 of a service's backends are available, the script
+is 50, meaning if less than 50% of a service's backends are available, the script
```

Comment:
The default is still 50%: https://github.com/Yelp/paasta/blob/move-replication-monitoring-to-config/paasta_tools/marathon_tools.py#L525-L526 

Should this be removed? I'll clarify the language above either way